.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         otp_vsn: [21.3.8.17, 22.3.4.9, 23.1.5]
-        os: [ubuntu-latest]
```

Comment:
^ What temporarily broke, specifically?

Going ahead with the merge, in any case.

Reply:
`Package libwxgtk3.0-0v5 is not available, but is referred to by another package.`